packages/hooks/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@4c/git-hooks",
+  "version": "1.0.0",
+  "main": "lib.js",
+  "repository": {
+    "directory": "packages/hooks",
+    "type": "git",
+    "url": "https://github.com/4Catalyzer/cli.git"
+  },
+  "bin": {
+    "git-hooks": "cli.js"
+  },
+  "author": "4Catalyzer",
+  "license": "MIT",
+  "scripts": {
+    "tdd": "jest --watch",
+    "_install": "node cli install",
+    "preuninstall": "node cli uninstall",
+    "prepublishOnly": "pinst --enable",
+    "postpublish": "pinst --disable",
```

Comment:
since it's super unclear, we want the `install` hook to exist for consumers of this package, but do not want it to run when we locally run `yarn` while working on this package, `pinst` edits the pkg.json and removes or adds the underscores when needed, e.g. right before we publish, it removes them, and then adds them back after publish